editor/src/clj/editor/gui.clj

```diff
@@ -3216,8 +3230,6 @@
   (input default-tex-params g/Any)
   (output default-tex-params g/Any (gu/passthrough default-tex-params))
   (input display-profiles g/Any)
-  (input current-layout g/Str)
-  (output current-layout g/Str (g/fnk [current-layout visible-layout] (or current-layout visible-layout)))
```

Comment:
Folded into `trivial-gui-scene-info`, where `:current-layout` is assigned the `visible-layout` of this `GuiSceneNode`. The `:current-layout` is then replaced with the value from the referencing scene when our scene is imported as a template.